api/odigos/v1alpha1/destination_types.go

```diff
@@ -62,3 +62,17 @@ type DestinationList struct {
 func init() {
 	SchemeBuilder.Register(&Destination{}, &DestinationList{})
 }
+
+/* Implement common.ExporterConfigurer */
+func (dest Destination) GetName() string {
+	return dest.Name
+}
```

Comment:
Destinations has 2 names: one for the CRD object itself in k8s (which is what we use here) and one free-style name that the user can fill which is found in `Spec`. 

I suggest stating which name is used so that it's not confusing in the future.